developer/src/kmcmplib/src/CompMsg.cpp

```diff
@@ -0,0 +1,129 @@
+#include <CompMsg.h>
+
+struct CompilerError {
+  KMX_DWORD ErrorCode;
+  const  KMX_CHAR* Text;
+  };
+
+const struct CompilerError CompilerErrors[] = {
+    { CERR_InvalidLayoutLine  , "Invalid 'layout' command"},
+    { CERR_NoVersionLine      , "No version line found for file"},
+    { CERR_InvalidGroupLine   , "Invalid 'group' command"},
+    { CERR_InvalidStoreLine   , "Invalid 'store' command"},
+    { CERR_InvalidCodeInKeyPartOfRule , "Invalid command or code found in key part of rule"},
+    { CERR_InvalidDeadkey     , "Invalid 'deadkey' or 'dk' command"},
+    { CERR_InvalidValue       , "Invalid value in exteded string"},
+    { CERR_ZeroLengthString   , "A string of zero characters was found"},
+    { CERR_TooManyIndexToKeyRefs  , "Too many index commands refering to key string"},
+    { CERR_UnterminatedString , "Unterminated string in line"},
+    { CERR_StringInVirtualKeySection  , "exted string illegal in virtual key section"},
+    { CERR_AnyInVirtualKeySection  , "'any' command is illegal in virtual key section"},
+    { CERR_InvalidAny         , "Invalid 'any' command"},
+    { CERR_StoreDoesNotExist  , "Store referenced does not exist"},
+    { CERR_BeepInVirtualKeySection  , "'beep' command is illegal in virtual key section"},
+    { CERR_IndexInVirtualKeySection  ,  "'index' command is illegal in virtual key section"},
+    { CERR_BadCallParams      , "CompileKeyboardFile was called with bad parameters"},
+    { CERR_InfileNotExist     , "Cannot find the input file"},
+    { CERR_CannotCreateOutfile, "Cannot open output file for writing"},
+    { CERR_UnableToWriteFully , "Unable to write the file completely"},
+    { CERR_CannotReadInfile   , "Cannot read the input file"},
+    { CERR_SomewhereIGotItWrong , "Internal error: contact Keyman"},
+    { CERR_BufferOverflow     , "The compiler memory buffer overflowed"},
+    { CERR_Break              , "Compiler interrupted by user"},
+    { CERR_CannotAllocateMemory , "Out of memory"},
+    { CERR_InvalidBitmapLine  , "Invalid 'bitmaps' command"},
+    { CERR_CannotReadBitmapFile , "Cannot open the bitmap or icon file for reading"},
+    { CERR_IndexDoesNotPointToAny  , "An index() in the output does not have a corresponding any() statement"},
+    { CERR_ReservedCharacter  , "A reserved character was found"},
+    { CERR_InvalidCharacter   , "A character was found that is outside the valid Unicode range (U+0000 - U+10FFFF)"},
+    { CERR_InvalidCall        , "The 'call' command is invalid"},
+    { CERR_CallInVirtualKeySection  , "'call' command is illegal in virtual key section"},
+    { CERR_CodeInvalidInKeyStore  , "The command is invalid inside a store that is used in a key part of the rule"},
+    { CERR_CannotLoadIncludeFile  , "Cannot load the included file: it is either invalid or does not exist"},
+    { CERR_60FeatureOnly_EthnologueCode  , "EthnologueCode system store requires VERSION 6.0"},
```

Comment:
Should the message be changed to "...requires VERSION 6.0 or higher" ? Same for  other strings with `VERSION` or `store(version)`.

Reply:
OK